app/models/concerns/password_validatable.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module PasswordValidatable
```

Comment:
The name of this module sounds like a polymorphic relationship. 
```suggestion
module PasswordValidator
```